README.md

```diff
@@ -123,23 +124,29 @@ You can...
 
 ## Build
 
----
----
+### Building
 
-You can **STOP READING** now unless you want to know how to build `forge` and make releases!
-
----
----
```

Comment:
I like to keep something like this in, because it happens so many times that people assume that you have to do these steps as well.

Reply:
Sure, although I expect that they might miss this as well if they miss the big "INSTALLATION" part at the top of the file :) I will revert this.